docs/src/main/asciidoc/security-testing.adoc

```diff
@@ -0,0 +1,79 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Security Testing
+
+include::./attributes.adoc[]
+
+This document describes how to test Quarkus Security.
+
+== Configuring User Information
+
+You can use link:security-properties[quarkus-elytron-security-properties-file] for testing security. This supports both embedding user info in `application.properties` and standalone properties files.
```

Comment:
I think more explaintation is needed.
The idea here is to use `quarkus-properties-files` for test and another one for prod right ?

Maybe provide an `application.properties` example with two security extension and the right config to enable one for test and the other for prod.

Reply:
@loicmathieu Sure, I've just moved this text to this new document. I think you use it in your tests, right ? We can just say, `for example, look how it has been done [there](link)`

Reply:
I don't have a link at the moment, but something like this can explain how it should be done.

```
# Configure embedded authentication for test
%test.quarkus.security.users.embedded.enabled=true
%test.quarkus.security.users.embedded.plain-text=true
%test.quarkus.security.users.embedded.users.scott=reader
%test.quarkus.security.users.embedded.users.stuart=writer
%test.quarkus.security.users.embedded.roles.scott=READER
%test.quarkus.security.users.embedded.roles.stuart=READER,WRITER

# Configure OAuth2
quarkus.oauth2.enabled=true
%test.quarkus.oauth2.enabled=false
# Other `quarkus.oauth2` configuration should be added bellow
quarkus.oauth2.client-id= ...
```

Reply:
I can look up as well